### DISCUSSION TITLE
Energy distribution dashboard how to?

### DISCUSSION THREAD

**Discussion by man55** [kellerza/sunsynk#315] [Q&A]
Hello,
first thanks for the work done.

I added my "Deye SUN-6K-SG03LP1-EU WiFi" to HA via solarman driver and mqqt in configuration, thanks to the YouTube video with step-by-step instructions. I now have sensors.

I would also like to add such a beautiful Energy distribution widget as in the image, but I don’t understand how to do it.
Is there a video or any step-by-step instructions on where I need to click and what to add?
Thank you.

![Снимок экрана 2024-07-10 120112](https://github.com/kellerza/sunsynk/assets/30772900/2b6e791e-f6d9-438c-8e6a-8680ce983bc5)

**Comment by kellerza** [comment]
It's a built-in HA function. There are links on that page where you can read more on it.

**Comment by man55** [comment]
Yeah, it's clear now. I thought it was a built-in widget. There is too much text on those links and, as always, with their instructions nothing is clear to the average person who is not a programmer.
I found a third-party dashboard, it was at least understandable and was able to automatically identify most entities.github.com/slipx06/sunsynk-power-flow-card.

Thanks for your integration.
I just have to figure out how to add the generator sensors, since the generator is connected to the GEN port on my Deye, I may have to open a new topic.

**Reply by kellerza** [reply]
Those dashboards serve a different purpose

the one is energy. Answering the question: how much did I use? (Or did I pay for/would have payed for, do I use too much power at night, deplete my batteries with some load that I should try move to the day, where I might have excess power, etc)

the other is power. Answering the question: how much power do I currently use? (In the current moment. Now. Is this in my inverter limits, is the geyser, kettle and stove on at the same time, can I distribute this usage better by switching off something now)

Glad you found it 😉https://kellerza.github.io/sunsynk/examples/lovelace
